types/spectest/tests/signedssvmsg/valid.go

```diff
@@ -0,0 +1,32 @@
+package signedssvmsg
+
+import (
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+// Valid tests a valid SignedSSVMessageTest
+func Valid() *SignedSSVMessageTest {
```

Comment:
Now that you added `MessageValidator` mayb sig tests should be in `MessageValidator`...

The issue is since our `MessageValidator` isn't aligned with implementation not sure how this will work out for the SSV team.

wdyt @y0sher and @moshe-blox

Reply:
I agree a lot has to be done regarding Message Validation. Plus, tests for it and SignedSSVMessage are related. But I think we could do it in a later PR, since (if I'm not mistaken) it's not a top priority and we're overheaded with other tasks.

Reply:
@GalRogozinski @MatheusFranco99 
I think this test should just use message validator as a blackbox, this means this test should check that all the behavior  we expect to happen or not happen based on the msg validator response should be tested under SignedSSVMsg tests, all the stuff regarding message validation (signatures and other validations) should happen on its own place.
For this test, what message validator did doesn't matter , it should just test what it expects to see if message validator returned valid/invalid. could use a mock imo.